docs/scripts.rst

```diff
@@ -130,6 +130,10 @@ Explanation for the keys:
 - ``urls``: dictionary of table names and the respective urls
 - ``version``: "1.0.0"
 
+``fields``: Although retriever automatically detects the schema of the table,
```

Comment:
consider the schema of table 
`
"schema": {
    "fields": [
         {
             "name": "tree_id",
              "type": "char"
              "size":"5"
         }
    ]
}                                
`

let us say we want to make tree-id as primary key. 
By mentioning `pk-auto` in data type while creating JSON file from CLI, does it will change the values of `tree_id` from `char` of size 5 to `int` starting from 1 ?

If yes, Do we need to mention this in docs as cautions?